src/v/v8_engine/tests/executor_test.cc

```diff
@@ -0,0 +1,178 @@
+/*
+ * Copyright 2021 Vectorized, Inc.
+ *
+ * Licensed as a Redpanda Enterprise file under the Redpanda Community
+ * License (the "License"); you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ *
+ * https://github.com/vectorizedio/redpanda/blob/master/licenses/rcl.md
+ */
+
+#include "seastarx.h"
+#include "v8_engine/executor.h"
+
+#include <seastar/core/sleep.hh>
+#include <seastar/core/smp.hh>
+#include <seastar/core/when_all.hh>
+#include <seastar/testing/thread_test_case.hh>
+
+#include <algorithm>
+
+SEASTAR_THREAD_TEST_CASE(task_queue_full_test) {
+    struct task_for_test {
+        void operator()() { sleep(1); }
+
+        void cancel(){};
+
+        void on_timeout() {}
+    };
+
+    v8_engine::executor test_executor(1, ss::smp::count);
```

Comment:
cpu_id = 1? will this test crash if run with `-c 1` ?

Reply:
ctest run this test with -c 1. It works.

Reply:
with -c1 the valid set of cpu-id's is {0} but it looks like you are referencing cpu-id 1 as the first parameter. can you fix that?

Reply:
cpu-id is for std::thread. Not for executor.

Reply:
Or, I can not use core-1 for std::thread? In test I want to have seastar on core-0 and std::thread on core-1